src/fides/api/models/detection_discovery.py

```diff
@@ -47,15 +58,117 @@ class MonitorConfig(Base):
         nullable=False,
         index=True,
     )
+    databases = Column(
+        ARRAY(String),
+        index=False,
+        unique=False,
+        nullable=False,
+        server_default="{}",
+        default=dict,
+    )  # the databases to which the monitor is scoped
+    monitor_execution_trigger = Column(
+        MutableDict.as_mutable(JSONB),
+        index=False,
+        unique=False,
+        nullable=True,
```

Comment:
This doesn't matter, but we don't really have to specify this nullable=True default or unique=False default - in some models we were only specifying non-default attributes.  Existing classify_parms already has this though

Reply:
yeah i always go a bit back and forth - to your point, maybe i'll lean more toward not specifying, sometimes specifying can be confusing in its own right! i guess i just usually forget whether nullable is True or False by default 😬 

(hence the back and forth 😆)